### DISCUSSION TITLE
Does this solve the MQTT spam issue?

### DISCUSSION THREAD

**Discussion by uniquePWD** [doctor64/tuyaZigbee#25] [General]
Honestly, I kinda went all in on Zigbee devices from AliExpress and most of those ended up being Tuya devices. Luckily though, I bought a good coordinator, because my days, Tuya software spams the hell out of Zigbee2MQTT.

I'm wondering, would replacing the firmware with this mean that my devices would implement Zigbee properly and not spam my network?

**Comment by doctor64** [comment]
Yes, sort of. All my firmwares conforms to ZCL standard for reporting configuration, so you can set you own how often data (like battery, etc) reported to coordinator. At least using Z2M. Unfortunately, i don't have ZHA and don't know if it provide similar functionality.

**Reply by uniquePWD** [reply]
Amazing, I can't wait to switch all my devices over.
